src/dynamic_graph/sot/talos/talos.py

```diff
@@ -46,6 +48,8 @@ class Talos(AbstractHumanoidRobot):
                                (0.,1.,0.,0.),
                                (0.,0.,1.,-0.107),
                                (0.,0.,0.,1.))
+    defaultFilename = "talos_reduced_v2.urdf"
```

Comment:
Didn't you want to point to *pyrene.urdf* ?

Reply:
You are right, I will push the modification.

Reply:
In fact no. A reduced model is expected here. I do not want to break existing application, but we need to read the full model from file in the future.